client/src/components/create-assignment/ConfirmationDialog.js

```diff
@@ -16,14 +16,14 @@ import Button from '@mui/material/Button';
 function ConfirmationDialog({ handleClose, open, removeAssignment, indices, assignments }) {
   return (
     <Dialog open={open} >
-      <DialogTitle >Delete sub-assignment</DialogTitle>
+      <DialogTitle >Delete Sub-Attainments</DialogTitle>
```

Comment:
Are we calling these sub-attainments or sub study attainments? There are some inconsistencies with this in Figma, so we should decide on one wording

Reply:
Decided with the frontend team to use 'sub study attainment' when there's room, so almost always, but in buttons and input labels we use 'sub attainment'.